### ISSUE TITLE
OpenEphys data support

### ISSUE THREAD

**Issue by mdiamantaki** [NeuroNetMem/PowerPixelsPipeline#1]
Hey, this pipeline looks great! I'm very eager to try it (and hopefully use it), but all my data are recorded with OpenEphys. Is there any chance you would implement a version that supports OpenEphys recorded data? Thanks!

**Comment by guidomeijer** [comment]
Yes I can try to implement Open Ephys support. However, because I'm using a lot of components from the IBL pipeline I don't know if all steps will work. The IBL uses NI hardware and some things (like synchronization between probes) are specific for the hardware. Could you maybe send me a (short) recording so that I can use it for testing? My email is guido.meijer@donders.ru.nl

**Comment by guidomeijer** [comment]
I've updated the pipeline and it now fully supports data recorded with the OpenEphys GUI.
